tests/integration/test_with_portainer_service.py

```diff
@@ -2,13 +2,57 @@
 # pylint: disable=unused-argument
 
 import asyncio
+import os
+import subprocess
+import time
 from collections.abc import AsyncIterator
+from pathlib import Path
+from typing import Any, Callable
 
 import pytest
 from aiohttp import ClientSession
+from faker import Faker
 from yarl import URL
 
-from simcore_service_deployment_agent import portainer
+import docker
+from simcore_service_deployment_agent import exceptions, portainer
+
+pytest_plugins = [
+    "pytest_simcore.docker_registry",
+    "pytest_simcore.docker_swarm",
+    "pytest_simcore.schemas",
+    "pytest_simcore.repository_paths",
+    "fixtures.fixture_portainer",
+]
+
+
+@pytest.fixture(scope="session")
+def osparc_simcore_root_dir(request) -> Path:
```

Comment:
suggestion: add a NOTE in comments that this is necessary to override base pytest-simcore fixtures.

Reply:
thx, done